src/servers/Scripts/quest/ManFst005.cpp

```diff
@@ -0,0 +1,213 @@
+                                    {},
+                                      eventId );
+      }
+   }
+
+   private:
+
+   void Scene00000( Entity::Player& player )
+   {
+      player.playScene( getId(), 0, HIDE_HOTBAR,
+                         [ & ]( Entity::Player& player, const Event::SceneResult& result )
+                         {
+                            if( result.param2 == 1 )
+                            {
+                               Scene00001( player );
+                            }
+                         } );
+   }
+
+   void Scene00001( Entity::Player& player )
+   {
+      player.playScene( getId(), 1, SET_EOBJ_BASE | HIDE_HOTBAR | INVIS_EOBJ,
+                         [ & ]( Entity::Player& player, const Event::SceneResult& result )
+                         {
+                            player.updateQuest( getId(), Seq1 );
+                         } );
+   }
+
+   void Scene00002( Entity::Player& player )
+   {
+      player.playScene( getId(), 2, HIDE_HOTBAR,
+                         [ & ]( Entity::Player& player, const Event::SceneResult& result )
+                         {
+                            Scene00003( player );
+                         } );
+   }
+
+   void Scene00003( Entity::Player& player )
+   {
+      player.playScene( getId(), 3, HIDE_HOTBAR,
+                         [ & ]( Entity::Player& player, const Event::SceneResult& result )
+                         {
+                            if( result.param2 == 1)
+                            {
+                               player.updateQuest( getId(), Seq3 );
+                               player.sendNotice( "QuestBattle actually disabled, goat said it...it's not my fault!" );
+                            }
+                         } );
+   }
+
+   void Scene00004( Entity::Player& player )
+   {
+      player.playScene( getId(), 4, HIDE_HOTBAR,
+                         [ & ]( Entity::Player& player, const Event::SceneResult& result )
+                         {
+                         } );
+   }
+
+   void Scene00005( Entity::Player& player )
+   {
+      player.playScene( getId(), 5, HIDE_HOTBAR,
+                         [ & ]( Entity::Player& player, const Event::SceneResult& result )
+                         {
+                            player.updateQuest( getId(), Seq3 );
+                         } );
+   }
+
+   void Scene00006( Entity::Player& player )
+   {
+      player.playScene( getId(), 6, HIDE_HOTBAR,
+                         [ & ]( Entity::Player& player, const Event::SceneResult& result )
+                         {
+                            Scene00007( player );
+                         } );
+   }
+
+   void Scene00007( Entity::Player& player )
+   {
+      player.playScene( getId(), 7, HIDE_HOTBAR,
+                         [ & ]( Entity::Player& player, const Event::SceneResult& result )
+                         {
+                            player.sendQuestMessage( getId(), 2, 1, 30426, 3750473944 );
```

Comment:
these values seem pretty arbitrary, what do they actually mean? Especially the 2nd one...